### ISSUE TITLE
`brew install` not working when older fishtown tap is known

### ISSUE THREAD

**Issue by bleonard** [dbt-labs/dbt-core#3860]
I have don `brew update` but the older `fishtown-analytics` taps seem to remain.

Just doing `brew install dbt` get this:
```
➜  ~ brew install dbt
Error: Formulae found in multiple taps: 
       * dbt-labs/dbt/dbt
       * fishtown-analytics/dbt/dbt
```

But when I do the fully qualified name, there is still an issue.
```
➜  ~ brew install dbt-labs/dbt/dbt
==> Installing dbt from dbt-labs/dbt
==> Downloading https://ghcr.io/v2/homebrew/core/openssl/1.1/manifests/1.1.1l
....
==> /opt/homebrew/Cellar/dbt/0.20.1_1/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /priv
Error: Formulae found in multiple taps: 
       * dbt-labs/dbt/dbt@0.12.2-rc1
       * fishtown-analytics/dbt/dbt@0.12.2-rc1

Please use the fully-qualified name (e.g. dbt-labs/dbt/dbt@0.12.2-rc1) to refer to the formula.
```

Maybe somewhere inside the tap is a reference that is not qualified?

I suppose, it's also odd that the old version (`0.12.2`) is referenced somewhere at all.

**Comment by bleonard** [comment]
Trying to run `brew install dbt` again after "forgetting" about the old taps I must have added in the past.
```
➜  ~ brew untap fishtown-analytics/dbt
```

It seemed to think `dbt` was done even though it had exited early before:
```
➜  ~ brew uninstall dbt
```

Ok, trying again:
```
➜  ~ brew install dbt  
==> Installing dbt from dbt-labs/dbt
==> Downloading https://files.pythonhosted.org/packages/05/eb/9c72cb72bdbe8652f485b993eec7c7fb31e721310b03ccf7f95a0ae57c61/dbt-bigquery-0.20.1.tar.gz
...
==> /opt/homebrew/Cellar/dbt/0.20.1_1/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/dbt-20210905-86141-14fpn72/dbt-0.20.1
🍺  /opt/homebrew/Cellar/dbt/0.20.1_1: 9,866 files, 161.9MB, built in 7 minutes 59 seconds
==> `brew cleanup` has not been run in 30 days, running now...
Removing: /opt/homebrew/Cellar/gdbm/1.18.1_1... (25 files, 876.2KB)
Removing: /Users/brian/Library/Caches/Homebrew/heroku--7.56.1.tar.xz... (7MB)
Removing: /opt/homebrew/Cellar/icu4c/67.1... (258 files, 72.4MB)
Removing: /opt/homebrew/Cellar/krb5/1.18.3... (162 files, 5.9MB)
Removing: /opt/homebrew/Cellar/krb5/1.19... (162 files, 5.9MB)
Removing: /opt/homebrew/Cellar/libffi/3.3_2... (17 files, 617.4KB)
Removing: /opt/homebrew/Cellar/libssh2/1.9.0_1... (184 files, 997.6KB)
Removing: /opt/homebrew/Cellar/openssl@1.1/1.1.1i... (8,067 files, 18MB)
Removing: /opt/homebrew/Cellar/postgresql/13.1... (3,217 files, 43.4MB)
Removing: /opt/homebrew/Cellar/python@3.8/3.8.7_2... (4,499 files, 73MB)
Removing: /opt/homebrew/Cellar/rust/1.49.0_1... (15,820 files, 578MB)
Removing: /opt/homebrew/Cellar/sqlite/3.34.0... (11 files, 4.2MB)
Removing: /Users/brian/Library/Caches/Homebrew/cargo_cache... (12,814 files, 303.6MB)
Removing: /Users/brian/Library/Logs/Homebrew/heroku... (114B)
Removing: /Users/brian/Library/Logs/Homebrew/heroku-node... (119B)
Pruned 0 symbolic links and 2 directories from /opt/homebrew
➜  ~ which dbt

/opt/homebrew/bin/dbt
```

success!

**Comment by jtcohen6** [comment]
@bleonard Thanks for opening! Totally get the confusion here; if you've previously tapped `fishtown-analytics/dbt`, I do recommend taking some time—seriously, lunch or an extended coffee break—to run:
```
brew untap fishtown-analytics/dbt --force
brew tap dbt-labs/dbt
brew install dbt
```

That should re-point your local installation from the formulae at https://github.com/fishtown-analytics/homebrew-dbt to those at https://github.com/dbt-labs/homebrew-dbt, and enable you to run:
- `brew upgrade dbt` as future (final) versions are released
- `brew install dbt@x.y.z-pre` as we push prereleases live

This should be quite straightforward for any totally new users of dbt + Homebrew, following the [steps we've documented](https://docs.getdbt.com/dbt-cli/installation#homebrew)https://docs.getdbt.com/dbt-cli/installation#homebrew). I don't know if there's a way we can ease this transition for existing Homebrew + dbt users.

FYI We're planning to rework that way that dbt is distributed and installed quite soon (#3361). I'm going to hold off on updating those installation docs until we've carried out those changes. When I do, I'll be sure to include a note regarding folks who had previously tapped `fistown-analytics/dbt`.
